lib/translator/common.h

```diff
@@ -30,6 +38,9 @@ enum class StatusCode {
 
 void DebugLog(const char* format, ...);
 
+void SetDebugCallback(void (*callback)(const char*));
+
+absl::string_view ScsiOpcodeToString(scsi_defs::OpCode opcode);
```

Comment:
nit: seems unused in PR at hand, maybe appropriate to have in a separate PR

Reply:
this branch originally was based off of master, now it is on gen-translation
perhaps the issue is gen-translation should pull from master?

Reply:
I was just refering to the ScsiOpcodeToString being unuserd

Reply:
ah sounds good